src/components/Results/ArticleSummary.js

```diff
@@ -93,9 +86,11 @@ function ArticleSummary(props) {
         <Grid item xs={12} md={8}>
           <Typography variant="body1">
             Keywords:
-            {keywordArr.map((word) => (
-              
-              <Chip label={word.charAt(0).toUpperCase() + word.slice(1)} className={classes.keywordChip} />
+            {keywords.map((word) => (
```

Comment:
Does this work if there are no keywords, but there's no error?

Reply:
It's one of those lovely Jest test things we need to do I think (nervous breakdown)